src/pages/library/common.js

```diff
@@ -205,6 +206,7 @@ export const SearchAndFilterComponent = ({ featuredList, sidebarSections, active
   const [searchFilter, setSearchFilter] = useState('')
   const [sort, setSort] = useState({ field: 'created', direction: 'desc' })
   const [requestDatasetAccessList, setRequestDatasetAccessList] = useState()
+  const [datasetName] = useState('')
```

Comment:
Mind sticking the setter on here for now, even if it's not yet used? Just so people don't get confused.
```suggestion
  const [datasetName, setDatasetName] = useState('')
```